builder/src/version.rs

```diff
@@ -0,0 +1,15 @@
+// Licensed under the Apache-2.0 license
+
+pub const ROM_VERSION_MAJOR: u16 = 1;
```

Comment:
Do we want this in the source code, or fill this as part of the release process?

Reply:
This is something I am looking for input on. If you have an idea for a way we can populate this during a release I think that would be better. You could in theory try to pull it from a tag but I was thinking that may be fragile and requires the tag to be populated before the release is made.

Reply:
In the interest of getting this patch out quickly, I suggest we leave it like this right now, and look for a better method for the next release